src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -25,8 +25,8 @@ public static class StandardTrainersCatalog
         /// <param name="featureColumnName">The features, or independent variables.</param>
         /// <param name="exampleWeightColumnName">The name of the example weight column (optional).</param>
         /// <param name="numberOfIterations">The maximum number of passes through the training dataset; set to 1 to simulate online learning.</param>
-        /// <param name="initialLearningRate">The initial <a href="tmpurl_lr">learning rate</a> used by SGD.</param>
-        /// <param name="l2Regularization">The L2 weight for <a href='tmpurl_regularization'>regularization</a>.</param>
+        /// <param name="learningRate">The learning rate. Large value may boost the training process but also leads to numerical instability.</param>
```

Comment:
> boost the training process [](start = 73, length = 27)

"boost the training process" is unclear. Let's keep the `tmpurl` links in these. Shahab is adding these to all point to a common set of definitions for learning rate, L1, L2, etc.